src/error.rs

```diff
@@ -31,7 +31,7 @@ pub enum RoutingError {
     #[error(display = "Invalid State.")]
     Terminated,
     #[error(display = "Invalid requester or handler authorities.")]
```

Comment:
This description should probably be changed to "Invalid requester or handler locations". Also maybe look for any other instances of "authorities" (plural) and rename them as well.